assets/js/googlesitekit-datastore-site.js

```diff
@@ -20,6 +20,6 @@
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { registerStore } from 'assets/js/googlesitekit/datastore/site';
+import { registerStore } from './googlesitekit/datastore/site';
```

Comment:
I could add a resolver for this (like above) but it seems more sensible to just change the one reference!

Reply:
Yes, this is the correct change to make. Thanks!